phpstan.neon.dist

```diff
@@ -0,0 +1,4 @@
+parameters:
+    level: 5
```

Comment:
level 6 is failing due to missing strict type hinting on some method, ie in `MessageInterface`:
```php
    /**
     * @param mixed|null $default
     *
     * @return mixed|null
     */
    public function getMeta(string $key, $default = null);
```